README.md

```diff
@@ -2,7 +2,11 @@
 
 Packetgraph is a library aiming to give the user a tool to build networks graph easily, It's built upon the fast [DPDK library](http://dpdk.org/).
 
-[![Build Status](https://travis-ci.org/outscale/packetgraph.svg?branch=master)](https://travis-ci.org/outscale/packetgraph)
+\htmlonly
+<a href="https://travis-ci.org/outscale/packetgraph">
+<img src="https://travis-ci.org/outscale/packetgraph.svg">
+</a>
+\endhtmlonly
```

Comment:
I really don't like this solution.
Would it be possible to run a markdown -> html converter and pass the result to doxygen ?